### ISSUE TITLE
Rendering does not show original glyphs at any stage

### ISSUE THREAD

**Issue by sanketgarade** [simoncozens/crowbar#58]
I am a complete newbie to text rendering so forgive me if I use wrong terminology.

I tried to see the text `नमस्कार` using font [Noto serif Devanagari](https://fonts.google.com/noto/specimen/Noto+Serif+Devanagari).

I remember seeing about 2 weeks ago that at some of the shaping stage, if I moved my mouse on that stage the individual characters would show up in the render (in the top right corner) instead of the merged ligatures. But today (12 May) for all stages, the render remains unchanged.

i.e. -

previously : `नमस्कार` would show up something like -  `न म स ् क ा र` at some stages.

now : `नमस्कार` shows up like `नमस्कार` at all stages.

![image](https://github.com/simoncozens/crowbar/assets/74632379/851c7c7e-5c08-4bfd-b970-248d5827025b)

**Comment by simoncozens** [comment]
This is probably the same as #57. I'll take a look.

**Comment by simoncozens** [comment]
No, I see the problem; it's a different one. HarfbuzzJS implements the intermediate shaping by stopping once it sees "end lookup ...". But in this case, only one lookup application happens. We need to be able to stop it after the initial reordering as well.
